### PR TITLE
fix: filename wrap in status window

### PR DESCRIPTION
**Pull request by samwiseg0**
This issue has bothered me for some time and I figured I would attempt to fix it. If this is not the "best" way to do this please let me know. Tested in several browsers and iOS 15.

**Before the changes:**
Screen Shot 2022-02-27 at 8 07 15 PM
Screen Shot 2022-02-27 at 8 07 29 PM

**After the changes:**
Screen Shot 2022-02-27 at 8 04 53 PM
Screen Shot 2022-02-27 at 8 04 39 PM

Signed-off-by: Phillip Hernandez samwiseg00@gmail.com

### CODE REVIEW

File: src/components/widgets/status/StatusLabel.vue [hunk 1]
```diff
@@ -35,9 +35,9 @@ export default class AppSwitch extends Mixins(StateMixin) {
     > span {
       display: block;
       // white-space: nowrap;
-      text-overflow: ellipsis;
       overflow: hidden;
-      text-overflow: ellipsis;
+      word-break: break-all;
+      width: auto;
```
Comment (pedrolamas) [inline src/components/widgets/status/StatusLabel.vue:40]: I don't see this entry actually doing anything, what is the purpose of using `width: auto` here?

Comment (samwiseg0) [reply src/components/widgets/status/StatusLabel.vue]: You are correct. That was left over from my testing. I have removed it.

**Comment by pedrolamas** [comment]
Though I agree with you that the initial approach is quite flawed (and quite honestly, doesn't even seem to be doing a proper job), I am checking on the implications of such a change on a component that is used for more things than just displaying filenames!

Specifically, if there is an `M117` with a long text, it would also be truncated on letter instead of word, but that would look quite odd for that case... so a localized change for the filename only might make more sense here.

File: src/components/widgets/status/StatusLabel.vue (hunk 1 above)
Comment (pedrolamas) [reply src/components/widgets/status/StatusLabel.vue]: I was referring just to the `width` attribute, but I do agree that `text-overflow` is not working correctly here.

**Comment by samwiseg0** [comment]
> Though I agree with you that the initial approach is quite flawed (and quite honestly, doesn't even seem to be doing a proper job), I am checking on the implications of such a change on a component that is used for more things than just displaying filenames!
> 
> 
> 
> Specifically, if there is an `M117` with a long text, it would also be truncated on letter instead of word, but that would look quite odd for that case... so a localized change for the filename only might make more sense here.

That makes sense. I had not considered M117 as I do not use it. 

I am no css expert by any means and unsure of what would need to be done to make that specific to just the file name.

**Comment by pedrolamas** [comment]
Not only M117 apparently...

image

> I am no css expert by any means and unsure of what would need to be done to make that specific to just the file name.

I think I can help with that! 🙂

**Review by pedrolamas — APPROVED** [review]
LGTM

**Review by matmen — APPROVED** [review]

**Pull request merged**